source/api/csw.rst

```diff
@@ -2,3 +2,61 @@
 
 Catalog Service for the Web (CSW)
 #################################
+
+The CSW end point exposes the metadata records in your catalog in XML format using the OGC CSW protocol (version 2.0.2). 
+
+Two protocols are available:
+
+ - CSW: Provides the ability to search and publish metadata for data, services and related information.
+ - CSW-T: Provides an interface for creating, modifying and deleting catalog records via the CSW protocol.
+
+Configuration
+=============
```

Comment:
Can be merged with https://geonetwork-opensource.org/manuals/3.8.x/en/administrator-guide/configuring-the-catalog/csw-configuration.html ? and link the 2 pages ?

and the API page contains URL and Request sections below + a link to https://www.opengeospatial.org/standards/cat.